pkg/autoupdate/tuf/autoupdate.go

```diff
@@ -24,14 +24,14 @@ import (
 	"github.com/theupdateframework/go-tuf/data"
 )
 
-//go:embed assets/tuf-dev/root.json
+//go:embed assets/tuf-kolide/root.json
 var rootJson []byte
 
 const (
-	DefaultTufServer      = "https://tuf-devel.kolide.com"
+	DefaultTufServer      = "https://tuf.kolide.com"
 	AutoupdateErrorBucket = "tuf_autoupdate_errors"
 	defaultChannel        = "stable"
-	tufDirectoryName      = "tuf-dev"
+	tufDirectoryName      = "tuf-kolide"
```

Comment:
`tuf-kolide` vs `tuf`? (ditto for the embed?)

Reply:
I can do just `tuf`! I'll update